src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -814,7 +814,7 @@ export class zonePacketHandlers {
         server.setGodMode(client, false);
         client.character.tempGodMode = false;
       }
-      client.character.positionUpdate = packet.data;
+      //client.character.positionUpdate = packet.data;
```

Comment:
if we remove it, we should remove the usage of it when sending LightweightToFullPc